packages/office-ui-fabric-react/src/components/SpinButton/examples/SpinButton.CustomStyled.Example.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { SpinButton, ISpinButtonStyles } from 'office-ui-fabric-react/lib/SpinButton';
+import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
+
+export class SpinButtonCustomStyledExample extends React.Component<any, any> {
+  public render() {
+    const styles: Partial<ISpinButtonStyles> = {
+      container: {
+        width: '400px'
+      }
+    };
+
+    const upArrowButtonStyles: Partial<IButtonStyles> = {

```

Comment:
minor suggestion, move these above the class, so that they're not redefined on each render.